src/core_plugins/kibana/public/management/sections/objects/components/objects_table/objects_table.js

```diff
@@ -141,7 +138,8 @@ export class ObjectsTable extends Component {
     if (!activeQuery) {
       return {
         pageOfItems: [],
-        totalItemCount: 0,
+        // Deliberately don't reset totalItemCount because 0 wouldn't be accurate.
```

Comment:
Why does this method short circuit when there are not active queries?

Reply:
I'm not sure and looking through the code I actually don't see how this branch could ever be hit. I'm going to try removing it.